contract/contract.go

```diff
@@ -306,6 +333,49 @@ func (c *Contract) GetTransTokenInfo() []TokenInfo {
 
 }
 
+func (c *Contract) GetCommitedTokensInfo() []TokenInfo {
```

Comment:
Similar to GetTransTokenInfo() . Can this function be moved to a generic utils ?

Reply:
TransTokenInfo info lie with in a separate nested block, where as commited tokens info is mostly present in gensys block, hence written as a separate function

Reply:
To be handled during [optimisation Milestone](https://github.com/rubixchain/rubixgoplatform/milestone/2)